### ISSUE TITLE
details about the implementation

### ISSUE THREAD

**Issue by chensnathan** [FengLi-ust/DN-DETR#6]
Hi, thanks for bringing new insights to the DETR series. DN-DETR is really an excellent work that can get such high performance with only 12 epochs.

After reading the paper, I have several questions about the detailed implementation of DN-DETR.
1. about the class embedding. According to the description of the class embedding in the paper and the discussion in the issue #3, the class embedding can be achieved by two different ways: (1) use a pre-trained language model to generate the embedding for the word of classes (classes of COCO with an unknown class: [person], [bicycle], [car], ..., [toothbrush], [unknown]); (2) use one-hot vector to represent different classes, then use Linear layer or MLP to project the one-hot vector to the latent space. Could you give more details about the implementation?
2. about the learning rate. I notice that DN-DETR uses an initial learning rate of 1e-5 with a batch size of 16 (Sec 5.1), which is different from the one in DAB-DETR(lr: 1e-4, lr_backbone: 1e-5 with a batch size of 16). Is it a typo or intended? If the learning rate is adjusted in DN-DETR, could you kindly report the gains of adjusting the learning rate?

Looking forward to a reply. Thanks in advance!

**Comment by FengLi-ust** [comment]
Thank you for pointing out the typo. For the first question, we adopt the second choice. For the second question, it is a typo. We will fix the typo and update our paper. Thank you.

**Comment by chensnathan** [comment]
Thanks for the clarification.

**Comment by yuanqianguang** [comment]
> Thank you for pointing out the typo. For the first question, we adopt the second choice. For the second question, it is a typo. We will fix the typo and update our paper. Thank you.

Hello, you said that Linear layer or MLP is used to project the one-hot vector to the latent space. Is it used to project the decoder content query for the consistency in the matching part? 
Wish for your reply.
